starter-code/pizza.js

```diff
@@ -1 +1,81 @@
-// Write your Pizza Builder JavaScript in this file.
+ // ITERATION 1
+
+// Add/remove pepperoni
+$('.btn-pepperoni.active').click(function(e) {
+  $('.pep').toggle();
+  $(this).toggleClass('active');
+  $('.panel.price ul li:nth(0)').toggle();
+  includedIngredients[0] = !includedIngredients[0];
+  updatePrice();
+});
+
+// Add/remove mushrooms
+$('.btn-mushrooms.active').click(function(e) {
+  $('.mushroom').toggle();
+  $(this).toggleClass('active');
+  $('.panel.price ul li:nth(1)').toggle();
+  includedIngredients[1] = !includedIngredients[1];
+  updatePrice();
+});
+
+// Add/remove mushrooms
+$('.btn-green-peppers.active').click(function(e) {
+  $('.green-pepper').toggle();
+  $(this).toggleClass('active');
+  $('.panel.price ul li:nth(2)').toggle();
+  includedIngredients[2] = !includedIngredients[2];
+  updatePrice();
+});
+
+//Add/remove white sauce
+$('.btn-sauce.active').click(function(e) {
+  $('.sauce').toggleClass('sauce-white');
+  $(this).toggleClass('active');
+  $('.panel.price ul li:nth(3)').toggle();
+  includedIngredients[3] = !includedIngredients[3];
+  updatePrice();
+});
+
+//Add/remove gluten free crust
+$('.btn-crust.active').click(function(e) {
+  $('.crust').toggleClass('active');
+  $(this).toggleClass('active');
+  $('.panel.price ul li:nth(4)').toggle();
```

Comment:
You should have added a class to the `li` element and also refactor the different functions because the code looks alike.
But good job 🙂